src/styles/global/tokens.scss

```diff
@@ -0,0 +1,266 @@
+  --fs-text-size-title-section      : var(--fs-text-size-4);
+  --fs-text-size-title-subsection   : var(--fs-text-size-4);
+  --fs-text-size-title-mini         : var(--fs-text-size-4);
+  --fs-text-size-lead               : var(--fs-text-size-4);
+  --fs-text-size-menu               : var(--fs-text-size-base);
+  --fs-text-size-body               : var(--fs-text-size-base);
+  --fs-text-size-legend             : var(--fs-text-size-1);
+  --fs-text-size-tiny               : var(--fs-text-size-0);
+}
+
+
+// --------------------------------------------------------
+// Spacing (UI Essentials)
+// --------------------------------------------------------
+
+body {
+  // NUMERIC SCALE
+  --fs-spacing-0                    : .25rem; //  4px
+  --fs-spacing-1                    : .5rem;  //  8px
+  --fs-spacing-2                    : .75rem; // 12px
+  --fs-spacing-3                    : 1rem;   // 16px (--fs-text-size-base)
+  --fs-spacing-4                    : 1.5rem; // 24px
+  --fs-spacing-5                    : 2rem;   // 32px
+  --fs-spacing-6                    : 2.5rem; // 40px
+  --fs-spacing-7                    : 3rem;   // 48px
+  --fs-spacing-8                    : 3.5rem; // 56px
+  --fs-spacing-9                    : 4rem;   // 64px
+  --fs-spacing-10                   : 4.5rem; // 72px
+  --fs-spacing-11                   : 5rem;   // 80px
+  --fs-spacing-12                   : 5.5rem; // 88px
+  --fs-spacing-13                   : 6rem;   // 96px
+}
+
+
+// --------------------------------------------------------
+// Grid & Layout (UI Essentials)
+// --------------------------------------------------------
+
+body {
+  // PADDING
+  --fs-grid-padding                 : var(--fs-spacing-3);
+  @include media(">=tablet") {
+    --fs-grid-padding               : var(--fs-spacing-4);
+  }
+  @include media(">=notebook") {
+    --fs-grid-padding               : var(--fs-spacing-5);
+  }
+
+  // CONTAINER
+  --fs-grid-max-width               : var(--fs-grid-breakpoint-notebook);
+
+  // GAPS
+  --fs-grid-gap-0                   : var(--fs-spacing-3);
+  --fs-grid-gap-1                   : var(--fs-spacing-4);
+  --fs-grid-gap-2                   : var(--fs-spacing-5);
+
+  // BREAKPOINTS
+  --fs-grid-breakpoint-phone        : #{map-get($breakpoints, "phone")};
+  --fs-grid-breakpoint-phonemid     : #{map-get($breakpoints, "phonemid")};
+  --fs-grid-breakpoint-tablet       : #{map-get($breakpoints, "tablet")};
+  --fs-grid-breakpoint-notebook     : #{map-get($breakpoints, "notebook")};
+  --fs-grid-breakpoint-desktop      : #{map-get($breakpoints, "desktop")};
+}
+
+
+// --------------------------------------------------------
+// Interactive Controls (UI Essentials)
+// --------------------------------------------------------
+
+body {
+  // TAP
+  --fs-control-tap-size-smallest    : calc(var(--fs-control-tap-size) / 2);
+  --fs-control-tap-size             : var(--fs-spacing-7); // 3rem = 48px
```

Comment:
Maybe we can add this [reference](https://web.dev/tap-targets/) here. Just to justify this number.

Reply:
Maybe we can use this approach: [Using media queries to detect touchscreen use](https://web.dev/accessible-tap-targets/#using-media-queries-to-detect-touchscreen-use) instead of checking only screen size here.

Reply:
Because this token has an impact on the appearance of the UI, I believe we should use the same media query type as other components to transition the layout.